web-components/src/components/cards/PostCard/PostCard.tsx

```diff
@@ -0,0 +1,331 @@
+import React from 'react';
+import { Box, Button, Grid } from '@mui/material';
+import { DefaultTheme as Theme } from '@mui/styles';
+import { makeStyles } from 'tss-react/mui';
+
+import { LockIcon } from '../../icons/LockIcon';
+import VerifiedIcon from '../../icons/VerifiedIcon';
+import WhitepaperIcon from '../../icons/WhitepaperIcon';
+import { Image, OptimizeImageProps } from '../../image';
+import StaticMap from '../../map/StaticMap';
+import { Body, Subtitle } from '../../typography';
+import UserInfo, { User } from '../../user/UserInfo';
+import Card from '../Card';
+import ActionButton from './PostCardActionButton';
+
+const formatDate = (dateString: string) => {
```

Comment:
originally I didn't use it because the default output didn't match the desired format in the designs, but I just noticed that `formatDate` takes a `format` string parameter and switched to use it now.